### ISSUE TITLE
All webapps - db connections need to be tested before used

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2235]
eg POSTing a context onto /mapfishapp/ws/wmc/ 

```json
{"success":false, "error":"fileupload_error_ioError", "msg": "org.postgresql.util.PSQLException: An I/O error occured while sending to the backend."}
```

Looks like the same issue we're trying to address with https://github.com/georchestra/georchestra/pull/2234

**Comment by fvanderbiest** [comment]
Check:
 * [ ] analytics
 * [ ] console
 * [ ] extractorapp
 * [ ] mapfishapp
 * [ ] security-proxy with ogc-service-statistics

**Comment by groldan** [comment]
Closing, fixed by #2436
